### PR TITLE
git_hub -> github

### PR DESCRIPTION
**Pull request by tarebyte**
I don't know why I never did this before, this adds the acronym `GitHub` so that we don't have to make file names like `git_hub` anymore.

/cc @johndbritton

### CODE REVIEW

File: config/initializers/inflectors.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+ActiveSupport::Inflector.inflections do |inflect|
+  inflect.acronym 'GitHub'
```
Comment (johndbritton) [inline config/initializers/inflectors.rb]: I didn't even know this was a thing. So good.

Comment (johndbritton) [inline config/initializers/inflectors.rb]: @mkcode FYI

**Comment by mkcode** [comment]
This is smart. I wonder if core/rails would accept a PR for this. Surely it more useful than 'sheep' :sheep: 

https://github.com/rails/rails/blob/master/activesupport/lib/active_support/inflections.rb

**Comment by mkcode** [comment]
Never mind. Core infections are frozen.

**Pull request merged**